frontend/app/src/components/home/profile/AccountsModal.svelte

```diff
@@ -11,7 +20,27 @@
         {$_("wallet")}
     </div>
     <div slot="body">
-        <Accounts />
+        <Accounts bind:showZeroBalance bind:zeroCount />
+    </div>
+    <div slot="footer">
+        <div class="footer">
+            <div class="show-more">
+                {#if zeroCount > 0}
```

Comment:
Shouldn't this 'if' check be before the "show_more" div?
Or is the div always rendered for some display reason?

Reply:
Yes it is supposed to be like that...